test/integration/tags/for.ts

```diff
@@ -366,4 +366,48 @@ describe('tags/for', function () {
       return expect(html).to.equal('i-someDrop i-someDrop i-someDrop ')
     })
   })
+
+  describe('parentloop', function () {
+    it('should handle no parent forloop', async function () {
+      const src = '{% for i in (1..2)%}{{ forloop.parentloop.index }}{% endfor %}'
+      const html = await liquid.parseAndRender(src, scope)
+      return expect(html).to.equal('')
```

Comment:
please treat this as a question: will this throw when `strict_variables` is set in shopify/liquid?

Reply:
Yes. I've added a test covering this case with `strictVariables` set to `true`.